src/engine/sync/internalclock.h

```diff
@@ -70,6 +70,7 @@ class InternalClock : public QObject, public Clock, public Syncable {
 
     int m_iOldSampleRate;
     double m_dOldBpm;
+    QAtomicInt m_bClockUpdated = false;
```

Comment:
`QAtomicInt` is equivalent to`QAtomicInteger<int>`. What about `QAtomicInteger<bool>`?